modules/genomic_browser/templates/menu_cpg_browser.tpl

```diff
@@ -0,0 +1,273 @@
+			</div>
+                        <label class="col-sm-12 col-md-2" data-toggle="tooltip" data-placement="top" title="Relation to UCSC CpG island">
+        		  {$form.Context.label}
+                        </label>
+                    	<div class="col-sm-12 col-md-2">
+        		  {$form.Context.html}
+			</div>
+		      </div>
+		    </div>
+		    <div class="row">
+                      <div class="form-group col-sm-12">
+                        <label class="col-sm-12 col-md-2">
+        		  {$form.Tissue.label}
+                        </label>
+                    	<div class="col-sm-12 col-md-2">
+        		  {$form.Tissue.html}
+			</div>
+                        <label class="col-sm-12 col-md-2">
+        		  {$form.Date_collected.label}
+                        </label>
+                    	<div class="col-sm-12 col-md-2">
+        		  {$form.Date_collected.html}
+			</div>
+                        <label class="col-sm-12 col-md-2">
+        		  {$form.Beta_value.label}
+                        </label>
+                    	<div class="col-sm-12 col-md-2">
+        		  {$form.Beta_value.html}
+			</div>
+		      </div>
+		    </div>
+                    <div class="row">
+                      <div class="form-group col-sm-12">
+                        <label class="col-sm-12 col-md-2">
+                          {$form.Platform.label}
+                        </label>
+                        <div class="col-sm-12 col-md-4">
+                          {$form.Platform.html}
+                        </div>
+                      </div>
+                    </div>
+	          </div> 
+                </div> <!--end of CpG filters panel-->
+              </div> 
+              <div class="form-group col-sm-4">
+                <div class="row"><!--fixed vertical space-->
+                  <br><br><br><br><br><br>
+                </div>
+                <div class="row">
+                  <div class="form-group col-sm-12">
+                    <label class="col-sm-4 col-md-3">
+                      {$form.Show_Brief_Results.label}
+                    </label>
+                    <div class="col-sm-8 col-md-8">
+                      {$form.Show_Brief_Results.html}
+                    </div>
+                  </div>
+                </div>
+                <div class="row">
+                  <div class="form-group col-sm-12">
+                    <div class="col-sm-6 col-xs-12 col-md-6">
+                      <input type="submit" name="filter" value="Show data" id="showdata" class="btn btn-sm btn-primary col-xs-12"/>
+                    </div>
+                    <div class="visible-xs col-xs-12"> </div>
+                    <div class="visible-xs col-xs-12"> </div>
+                    <div class="col-sm-6 col-xs-12 col-md-5">
+                      <input type="button" name="reset" value="Clear Form" class="btn btn-sm btn-primary col-xs-12" onclick="location.href='/genomic_browser/?reset=true'" />
```

Comment:
the link on this button should point to genomic_browser/cpg... not just genomic_browser/